__tests__/api.test.js

```diff
@@ -331,6 +331,37 @@ describe('DELETE /api/comments/:comment_id', () => {
     })
   });
 });
+
+describe('GET /api/users', () => {
+  test('200: responds with all available users', () => {
+      return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then(({ body })=> {
+          const { users }  = body;
```

Comment:
You can do nested destructuring if you would like to do this in one go. {body: {users}} would work the same too on line 340!

Reply:
thank you for the feedback! I will make the changes suggested